tests/41end-to-end-keys/07-backup.pl

```diff
@@ -40,6 +38,44 @@
       });
    };
 
+test "Responds correctly when backup is empty",
+   requires => [ $fixture, qw( can_create_backup_version ) ],
+
+   do => sub {
+      my ( $user ) = @_;
+      my $version;
+
+      matrix_get_key_backup_info( $user )->then( sub {
+         my ( $content ) = @_;
+
+         log_if_fail "Content", $content;
+
+         $version = $content->{version};
+
+         matrix_get_backup_key( $user, '!notaroom', 'notassession', $version);
+      })->main::expect_http_4xx
```

Comment:
why is this a 4xx rather than something more specific? Also, please can we have a comment to say what we are testing here?